packages/ember-testing/lib/test.js

```diff
@@ -464,6 +465,60 @@ EmberApplication.reopen({
       delete this.testHelpers[name];
       delete this.originalMethods[name];
     }
+  },
+
+  /**
+    Builds a new instance of the application and registers test helpers on it.
+
+    Example:
+
+    ```javascript
+    var instance = run(App, 'buildTestInstance');
+    ```
+
+    @public
+    @method buildTestInstance
+  */
+  buildTestInstance() {
+    var instance = this.buildInstance();
+    this.runInstanceInitializers(instance);
+    instance.injectTestHelpers();
+    return instance;
+  }
+});
+
+EmberApplicationInstance.reopen({
+  injectTestHelpers() {
```

Comment:
This function and `ApplicationInstance#removeTestHelpers` are super similar to the same functions on `Application`, I was unsure if I should extract them out though as they differ on a couple lines